dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -97,3 +97,14 @@ export type ValidatedPaymentDetailsFilters = {
   paymentCurrency?: Currency;
   paymentReference?: string;
 };
+
+export type FeeReportReconciledEmailVariables = {
+  bankRecipient: string;
+  reportReconciledDate: string;
+  reportPeriod: string;
+};
+
+export type FeeReportReconciledEmail = {
```

Comment:
Why are these prefixed by the word `Fee`? Should probably be `UtilisationReportReconciledEmail` or just `ReportReconciledEmail`